File: TestImport.java
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.Statement;
import java.nio.file.Files;
import java.nio.file.Paths;

public class TestImport {
    public static void main(String[] args) throws Exception {
        String url = "jdbc:mysql://grupoinnovate.co:3306/grupoin6_finixsoftware?useSSL=false&allowPublicKeyRetrieval=true";
        String user = "grupoin6_finixuser";
        String pass = "FinixPass2027...";
        
        System.out.println("Connecting to database...");
        try (Connection conn = DriverManager.getConnection(url, user, pass)) {
            System.out.println("Connected.");
            
            String content = new String(Files.readAllBytes(Paths.get("inserts_solamente.sql")));
            String[] queries = content.split(";");
            
            try (Statement stmt = conn.createStatement()) {
                for (String q : queries) {
                    q = q.trim();
                    if (q.isEmpty()) continue;
                    
                    try {
                        stmt.execute(q);
                    } catch (Exception e) {
                        System.out.println("\nERROR EXECUTING QUERY:");
                        System.out.println(q.substring(0, Math.min(q.length(), 200)) + "...");
                        System.out.println("\nEXCEPTION:");
                        e.printStackTrace();
                        System.exit(1);
                    }
                }
            }
            System.out.println("ALL EXECUTED SUCCESSFULLY!");
        }
    }
}

File: test_import.php
<?php
try {
    $pdo = new PDO('mysql:host=grupoinnovate.co;port=3306;dbname=grupoin6_finixsoftware', 'grupoin6_finixuser', 'FinixPass2027...', [
        PDO::ATTR_ERRMODE => PDO::ERRMODE_EXCEPTION
    ]);
    echo "Connected successfully.\n";
    
    $sql = file_get_contents('inserts_solamente.sql');
    $queries = explode(';', $sql);
    
    foreach ($queries as $q) {
        $q = trim($q);
        if (empty($q)) continue;
        
        try {
            $pdo->exec($q);
        } catch (PDOException $e) {
            echo "ERROR ON QUERY:\n";
            echo substr($q, 0, 200) . "...\n";
            echo "ERROR: " . $e->getMessage() . "\n";
            exit(1);
        }
    }
    echo "ALL QUERIES EXECUTED SUCCESSFULLY!\n";
} catch (PDOException $e) {
    echo "Connection failed: " . $e->getMessage();
}